python/mxnet/gluon/block.py

```diff
@@ -46,10 +47,10 @@ def __init__(self, block):
     @staticmethod
     def create(prefix, params, hint):
         """Creates prefix and params for new `Block`."""
-        current = _BlockScope._current
+        current = getattr(_BlockScope._current, "value", None)
```

Comment:
initialize _BlockScope._current.value to None globally and use _BlockScope._current.value directly here?